bigtable/client/internal/table_admin.cc

```diff
@@ -21,6 +21,11 @@ namespace bigtable {
 inline namespace BIGTABLE_CLIENT_NS {
 namespace noex {
 
+static_assert(std::is_copy_constructible<bigtable::noex::TableAdmin>::value,
+              "bigtable::noex::TableAdmin must be constructible");
+static_assert(std::is_copy_assignable<bigtable::noex::TableAdmin>::value,
```

Comment:
Here is the static_assert added

Reply:
Right, but that is for `bigtable::noex::TableAdmin`, I was asking for `bigtable::TableAdmin` without the `noex`.

Reply:
Sorry I misunderstood that. Now change it and commit.